File: web3/create.php
<!DOCTYPE html>
<html lang="en" dir="ltr">
  <head>
    <meta charset="utf-8">
    <title>I-ZONE INFORMATION REGISTER PAGE</title>
  </head>
  <body>
<h1>아이즈원 멤버 별 정보 등록 페이지</h1>

<ol>
  빈 칸에 값을 입력하세요
</ol>
<form  action="process_create.php" method="POST">
  <p> <input type="text" name="sex" placeholder="sex">        </p>
    <p> <input type="text" name="nickname" placeholder="nickname">        </p>
        <p> <input type="text" name="reason" placeholder="reason">        </p>
          <p> <input type="text" name="profile" placeholder="profile">        </p>
            <p> <input type="text" name="name" placeholder="name">        </p>
<!-- <p><textarea name="description" rows="8" cols="80" placeholder="description"></textarea> </p> -->
<p> <input type="submit"> </p>
</form>


  </body>
</html>

File: web2-python/create.py
#!C:\Users\ericshim\AppData\Local\Programs\Python\Python38\python.exe
print("Content-Type: text/html")
print()
import cgi, os,view






form = cgi.FieldStorage()
if 'id' in form:
    pageId = form['id'].value
    description=open('data/'+pageId,'r').read()
else:
    pageId="welcome"
    description='hello wiz*one'


print('''<!DOCTYPE html>
<html>
  <head>
    <meta charset="EUC-KR">
    <title>izone</title>
  </head>
  <body>
    <h1>         <a href="index.py">IZONE</a>         </h1>
    <ol>
{listStr}
    </ol>

<form action  ="process_create.py" method="post">

<a href="create.py">create</a>
<p><input type="text" name="title" placeholder="title"></p>
<p><textarea rows="5" name="description" placeholder="description" ></textarea></p>
<p><input type="submit"></p>

</form>

<h2>{title}</h2>
<p>{desc}</p>
  </body>
</html>




'''.format(title=pageId, desc=description,listStr=view.getlist()))
